tests/CodeGen-MkBundle/Xamarin.Android.MakeBundle-UnitTests/Xamarin.Android.MakeBundle-UnitTests.csproj

```diff
@@ -1,48 +1,21 @@
-<?xml version="1.0" encoding="utf-8"?>
-<Project DefaultTargets="Build" ToolsVersion="4.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+<Project Sdk="Microsoft.NET.Sdk">
+
   <PropertyGroup>
-    <Configuration Condition=" '$(Configuration)' == '' ">Debug</Configuration>
-    <Platform Condition=" '$(Platform)' == '' ">AnyCPU</Platform>
-    <ProjectGuid>{FA8EEC88-CA3C-4D69-B206-54B392570DC6}</ProjectGuid>
-    <OutputType>Library</OutputType>
+    <TargetFramework>net472</TargetFramework>
```

Comment:
Is there any value to making all these test projects `netstandard2.0`? Like maybe it would enable us to run `dotnet test` easily in the future?

Reply:
Unfortunately NUnit.ConsoleRunner is not compatible with netstandard, so we'd need to rewrite all of our runner logic to make that change.